pkg/eks/tasks.go

```diff
@@ -212,22 +239,26 @@ func (c *ClusterProvider) CreateExtraClusterConfigTasks(cfg *api.ClusterConfig,
 }
 
 // ClusterTasksForNodeGroups returns all tasks dependent on node groups
-func (c *ClusterProvider) ClusterTasksForNodeGroups(cfg *api.ClusterConfig, installNeuronDevicePluginParam bool) *manager.TaskTree {
+func (c *ClusterProvider) ClusterTasksForNodeGroups(cfg *api.ClusterConfig, installNeuronDevicePluginParam, installNvidiaDevicePluginParam bool) *manager.TaskTree {
 	tasks := &manager.TaskTree{
 		Parallel:  false,
-		IsSubTask: true,
+		IsSubTask: false,
 	}
 	var reallyInstallNeuronDevicePlugin bool
 	for _, ng := range cfg.NodeGroups {
 		reallyInstallNeuronDevicePlugin = reallyInstallNeuronDevicePlugin || api.HasInstanceType(ng, utils.IsInferentiaInstanceType)
 	}
+	var reallyInstallNvidiaDevicePlugin bool
```

Comment:
could we rename this? having `reallyInstallNeuronDevicePlugin` and `installNeuronDevicePluginParam` is confusing

Reply:
done